runtime/gc_structs/ConstantPoolObjectSlotIterator.hpp

```diff
@@ -48,18 +49,52 @@ class GC_ConstantPoolObjectSlotIterator
 	U_32 *_cpDescriptionSlots;
 	U_32 _cpDescription;
 	UDATA _cpDescriptionIndex;
+	bool _condyOnly;
+	bool _condyEnabled;
+	bool _isPreviousCondy;
 	
+private:
+	/**
+	 * Scan the current constant dynamic slot
+	 * @return slot value if it is not NULL
+	 * @return exception otherwise
+	 */
+	MMINLINE j9object_t* scanCondySlot(j9object_t *slotPtr)
```

Comment:
add an output parameter for this function that returns if it is moving to the next slot or not